src/VisualStudio/Core/Impl/Options/AbstractSettingsManagerOptionSerializer.cs

```diff
@@ -27,16 +27,15 @@ private class SVsSettingsPersistenceManager { };
         protected readonly ISettingsManager Manager;
         private readonly IOptionService _optionService;
 
-        public AbstractSettingsManagerOptionSerializer(IServiceProvider serviceProvider, IOptionService optionService)
+        public AbstractSettingsManagerOptionSerializer(VisualStudioWorkspaceImpl workspace)
```

Comment:
Could you be more specific.  What bug do you think is here?  It's not clear to me how/why the misc files workspace would interact with options.